pkg/testgridanalysis/testreportconversion/test_report.go

```diff
@@ -96,6 +104,35 @@ func PrepareTestReport(
 	return testReport
 }
 
+func getAllJobRunPassPercentage(allJobs []sippyprocessingv1.JobResult) sippyprocessingv1.TestResult {
+	runCount := 0
+	success := 0
+	failure := 0
+
+	for _, job := range allJobs {
+		if isNeverStable(job) {
+			continue
+		}
+
+		for _, runs := range job.AllRuns {
+			runCount++
+			if runs.OverallResult == sippyprocessingv1.JobSucceeded {
+				success++
+			} else {
+				failure++
```

Comment:
since this doesn't seem to be excluding job runs where we never even launched the e2e container/step, i think we need a different name for what this represents.

it also doesn't exclude job runs where the e2e step succeeded, but something went wrong during teardown/must-gather.

It's really just the overall pass rate of job runs across the system (Excluding never-stable)?

Reply:
I think you're right.  I could fix the name, or just take it out. I'm not actually displaying the data anywhere since I didn't think the number told me anything useful.

Reply:
I removed this